### PR TITLE
TAJO-1301: Support CREATE INDEX [index_name] on [table_name] ( [index_specs] ) LOCATION [PATH] statement

### PR DESCRIPTION
**Pull request by jihoonson**

### CODE REVIEW

File: tajo-core/src/test/java/org/apache/tajo/engine/query/TestCreateIndex.java [hunk 1]
```diff
@@ -95,4 +95,19 @@ public final void testCreateIndexOnMultiExprs() throws Exception {
...
   }
+
+  @Test
+  public final void testCreateIndexOnLocation() throws Exception {
+    executeQuery();
+    assertTrue(catalog.existIndexByName(getCurrentDatabase(), "l_orderkey_idx"));
+    assertTrue(catalog.existIndexByColumnNames(getCurrentDatabase(), "lineitem", new String[]{"l_orderkey"}));
+    catalog.dropIndex(getCurrentDatabase(), "l_orderkey_idx");
+    assertFalse(catalog.existIndexByName(getCurrentDatabase(), "l_orderkey_idx"));
+    executeString("create index l_orderkey_idx on lineitem (l_orderkey asc null first) location '/tajo/warehouse/default/l_orderkey_idx';");
```
Comment (jinossy) [inline tajo-core/src/test/java/org/apache/tajo/engine/query/TestCreateIndex.java]: Could you check the title and test case?  syntax is difference

Comment (jihoonson) [inline tajo-core/src/test/java/org/apache/tajo/engine/query/TestCreateIndex.java]: Thanks. I changed the title.

**Comment by jinossy** [comment]
+1 Looks great to me
Thanks

**Comment by jihoonson** [comment]
Thanks!
I committed to the index_support branch.

**Pull request closed without merging**